jax/experimental/jax2tf/jax2tf.py

```diff
@@ -800,8 +800,8 @@ def _select_and_gather_add(tangents: TfVal,
                            padding: Sequence[Tuple[int, int]]):
   # Note: this function follows the pattern in
   # jax.lax._select_and_gather_add_translation.
-  dtype = to_jax_dtype(operand.dtype)
-  nbits = dtypes.finfo(dtype).bits
+  dtype = to_tf_dtype(operand.dtype)
```

Comment:
I am not sure I understand this change. `operand` is a TF value, its `dtype` is a TF dtype. Why does it need to be used with `to_tf_dtype`?

Reply:
Oops – you are right. It's a noop for TF dtypes. Removed in latest commit.